torchtitan/parallelisms/parallelize_llama.py

```diff
@@ -441,9 +441,10 @@ def apply_compile(model, job_config: JobConfig):
     ac_config = job_config.activation_checkpoint
     if ac_config.mode == "selective" and ac_config.selective_ac_option == "op":
         # some temp flags for torch.compile enablement + SAC
-        torch._dynamo.config._experimental_support_context_fn_in_torch_utils_checkpoint = (
-            True
-        )
+        # torch._dynamo.config._experimental_support_context_fn_in_torch_utils_checkpoint = (
```

Comment:
this is required with recent versions of PyTorch as `torch._dynamo.config._experimental_support_context_fn_in_torch_utils_checkpoint` was recently deleted, is it ok if I just remove this code?  I can also leave as is.

Reply:
I think we can delete this after a few nightly versions removed this, but let's make it a separate PR as it's not related to fp8 changes?

Reply:
will do